zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterSetting.java

```diff
@@ -665,6 +660,13 @@ public void clearNoteIdAndParaMap() {
     runtimeInfosToBeCleared = null;
   }
 
+  public String getLauncherPlugin() {
```

Comment:
Cool!  Do you have any idea how can we make InterpreterLauncher user configurable?

Reply:
I was thinking to make it configurable in `interpreter-setting.json` and user could also customize it in UI. But I havn't nailed down the final design. Would do it in a separated PR after I complete the whole design.

Reply:
group isn't guaranteed to be "spark" right?

Reply:
It is always `spark` if it is spark interpreter even when there's multiple spark interpreters. We use field `name` to differentiate them